addon/helpers/hook.js

```diff
@@ -6,10 +6,38 @@ import returnWhenTesting from 'ember-hook/utils/return-when-testing';
 
 const { Helper } = Ember;
 
-export default Helper.extend({
-  compute(params, qualifiers = {}) {
-    const [hook] = params;
+/**
+ * Test if an object is empty `Ember.isEmpty` surprisingly doesn't do that
+ * @see {@link https://github.com/emberjs/ember.js/issues/5543}
+ * @see {@link https://github.com/emberjs/ember.js/blob/v2.10.0/packages/ember-metal/lib/is_empty.js}
+ *
+ * @param {Object} obj - the object to test
+ * @returns {Boolean} true if there are no keys on the object
+ */
+function isEmpty (obj) {
+  // Ember.keys is being deprecated in favor of Object.keys, handle both cases
+  if (Ember.keys) {
```

Comment:
I thought you might want to support older versions of `Ember`. If it were my project, I certainly wouldn't care, but I didn't want to make that assumption. I'll remove the check.